packages/app/src/queries/staking/utils.ts

```diff
@@ -59,8 +62,15 @@ export function getApy(totalStakedBalance: number, weekCounter: number) {
 
 export const parseEpochData = (index: number, networkId?: NetworkId) => {
 	const { epochStart, epochEnd } = getEpochDetails(networkId ?? 10, index)
-	const startDate = formatShortDate(new Date(toJSTimestamp(epochStart)))
-	const endDate = formatShortDate(new Date(toJSTimestamp(epochEnd)))
-	const label = `Epoch ${index}: ${startDate} - ${endDate}`
+	const label = `Epoch ${index}`
 	return { period: index, start: epochStart, end: epochEnd, label }
 }
+
+export const useEstimatedReward = (fileName: string) => {
+	const network = useAppSelector(selectNetwork)
+	const walletAddress = useAppSelector(selectWallet)
+	const query = useGetFile(
+		`trading-rewards-snapshots/${network === 420 ? 'goerli-' : ''}${fileName}`
+	)
+	return BigNumber.from(query?.data?.claims[walletAddress!]?.amount ?? 0)
```

Comment:
Might be better to use Wei to remain consistent with the rest of the app.

Reply:
Yes. Updated the code in https://github.com/Kwenta/kwenta/pull/2515/commits/26bf8ce8d171485eff34fe1811b3674450b575cc# `packages/sdk/src/services/kwentaToken.ts`.